### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
Description : 
1) user selectable option to show/ hide empty fields.
2) ignores display conditions specified in MMD

Pmdb : https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=24626

### CODE REVIEW

File: widget/info.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "subTitle": "Fields Of Interest",
+    "version": "1.0.0",
+    "metadata": {
+        "size": "lg",
+        "description": "",
+        "pages": [
+            "View Panel"
+        ],
+        "certified": "No",
+        "publisher": "Community",
```
Comment (cs-suyog-jinturkar) [inline widget/info.json]: Certified yes and publisher will be Fortinet

Comment (cs-suyog-jinturkar) [reply widget/info.json]: Please also add desc

File: widget/info.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        "compatibility": [
+            "7.3.1"
+        ]
+    },
+    "development": true
```
Comment (cs-suyog-jinturkar) [inline widget/info.json]: IMO, this key will no longer needed

File: widget/view.controller.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+                })
+            })
+
+            var missingFields = [];
+            var excludeArray = [];
+            $scope.config.excludeFieldsArray.forEach(function (field, index) {
+                excludeArray.push(field.name);
+            })
+            for (const key in $scope.entity.fields) {
+                if (!selectedFields.includes(key) && !excludeArray.includes(key) && $scope.entity.fields[key].type !== 'manyToMany'
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js:62]: You will skip all M2M and O2M fields, right?

File: widget/info.json (hunk 2 above)
Comment (cs-pranali-chavan) [inline widget/info.json]: "development": true field not required
published_date" field required with latest timestamp

File: widget/info.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        "size": "lg",
+        "description": "",
+        "pages": [
+            "View Panel"
+        ],
+        "certified": "No",
+        "publisher": "Community",
+        "compatibility": [
+            "7.3.1"
+        ]
```
Comment (cs-pranali-chavan) [inline widget/info.json:21]: "help_online" parameter required to add read me file link

File: widget/view.controller.js [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+            }
+            configRows.forEach(function (row) {
+                var defaultColumnStyle = viewTemplate.getColumnStyle(row.columns.length);
+                row.columns.forEach(function (column) {
+                    column.style = defaultColumnStyle;
+                    column.fields = $scope.updateFieldValues(column.fields);
+                });
+            });
+            $scope.rows = configRows;
+            console.log(configRows);
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js]: remove console

File: widget/view.controller.js (hunk 3 above)
Comment (cs-yash-patodkar) [reply widget/view.controller.js:62]: yes

**Review by cs-pranali-chavan — APPROVED** [review]
Looks good!

File: docs/media/detailed-view.png
Comment (cs-suyog-jinturkar) [inline docs/media/detailed-view.png:1]: Outdates image. Please use latest one

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**